### ISSUE TITLE
Non Steam PC client - not working?

### ISSUE THREAD

**Issue by joyzer** [RoseTheFlower/UltrawideIndex#22]
Hello there!

Is there anyting one has to do in order to have it working?
I am running their own client - not Steam - and I don't see any difference.

Cheers!

**Comment by joyzer** [comment]
Ah sorry! Path of Exile.

**Comment by RoseTheFlower** [comment]
According to [this comment](https://www.reddit.com/r/pathofexile/comments/wt5dvt/comment/il3mz8e/?context=3), it should work for the standalone launcher too. Did you see your resolution in the settings after dropping the fix files?

**Comment by joyzer** [comment]
I did but having black bars on either side.

**Comment by freddiehaddad** [comment]
Weird. For me, this works fine with the native client but not with the steam client.  With the steam client, regardless of the resolution, I still have black bars on my 5120x1440 ultrawide.  The renderer is on vulkan. Not sure if that matters.

**Comment by RoseTheFlower** [comment]
I re-test after most updates at 48:9 and I tested again at 3840x1080 on Steam today. No issues on Windows with either Vulkan or DX11. For Linux, the general thing to do is ensure that the DLL name is set to native and builtin in Wine.

**Comment by freddiehaddad** [comment]
Are you saying to rename the dll from version.dll to native.dll?


Freddie F. Haddad
619-269-8901

________________________________

**Comment by RoseTheFlower** [comment]
> Two questions: 1. Are you saying to rename the dll from version.dll to native.dll?

No, it's an [override to set via Wine configuration](https://i.imgur.com/d9Slz03.png).

**Comment by freddiehaddad** [comment]
Ah, thank you! That makes sense and is similar to this thread: https://www.reddit.com/r/linux_gaming/comments/ycunaj/cannot_get_the_ultrawide_fix_in_gotham_knights_to/

I know this is out of scope for this repository.  But how do you get to that window when all the Wine stuff is managed by Steam?

**Comment by RoseTheFlower** [comment]
> I know this is out of scope for this repository. But how do you get to that window when all the Wine stuff is managed by Steam?

I do not have a Linux repo installed at this time, so I wouldn't really know. It's been a while since I ran it as my main OS and a lot has changed. It's best to ask the people in that thread or sub.

**Comment by freddiehaddad** [comment]
Got it! There is a section in the configuration options for the game: [https://ibb.co/QJGLrqG](https://ibb.co/QJGLrqG)

In there I added:

WINEDLLOVERRIDES="version.dll=n,b" %command%

It's working!
